toonz/sources/tnztools/filltool.cpp

```diff
@@ -2032,11 +1932,26 @@ bool FillTool::onPropertyChanged(std::string propertyName) {
     if (m_segment.getValue()) FillType = ::to_string(m_fillType.getValue());
     FillSegment                        = (int)(m_segment.getValue());
   }
+
   // Autopaint
   else if (propertyName == m_autopaintLines.getName()) {
     rectPropChangedflag = true;
   }
 
+  else if (propertyName == m_maxGapDistance.getName()) {
+    if (TVectorImageP vi = getImage(true)) {
+      if (m_maxGapDistance.getValue() != vi->getAutocloseTolerance()) {
+        vi->setAutocloseTolerance(m_maxGapDistance.getValue());
+        int count = vi->getStrokeCount();
+        std::vector<int> v(count);
+        int i;
+        for (i = 0; i < (int)count; i++) v[i] = i;
+        vi->notifyChangedStrokes(v, std::vector<TStroke *>(), false);
+        if (m_level) m_level->setDirtyFlag(true);
```

Comment:
How about calling `TXshLevelHandle::notifyLevelChange()` as follows in order to update the Level Strip title bar?:
```
--------------------- toonz/sources/tnztools/filltool.cpp ---------------------
index e7a4095..67bfc52 100644
@@ -1947,7 +1947,10 @@ bool FillTool::onPropertyChanged(std::string propertyName) {
         int i;
         for (i = 0; i < (int)count; i++) v[i] = i;
         vi->notifyChangedStrokes(v, std::vector<TStroke *>(), false);
-        if (m_level) m_level->setDirtyFlag(true);
+        if (m_level) {
+          m_level->setDirtyFlag(true);
+          TTool::getApplication()->getCurrentLevel()->notifyLevelChange();
+        }
       }
     }
   }
```
Also, IMO it should have undo for modifying this value as it directly modifies the level data.
( The undo should be stored only on mouse release, but I'm not yet sure how to. )